index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>OAT | Portfolio</title>
+  <link rel="shortcut icon" href="./images/icons/fav.png" type="image/x-icon">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+      <a href="index.html">Oluwatoyin Olaoye</a>
+
+      <nav class="nav-link">
+        <ul class="link">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+        <a href="mailto:mindrepopps@gmail.com"><img src="./images/icons/mail_box.svg" alt="email destination"></a>
+      </nav>
+
+      <img src="./images/icons/menu_icon.png" alt="mobile menu bar">   
+  </header>
+
+  <section class="landing-page">
+    <div class="container">
+      <div class="main-intro">
+        <h1 class="theme" title="I am Oluwatoyin Olaoye">MindPrepDev</h1>
+        <h3 class="sub-theme">Software Developer</h3>
+        <p class="text">
+          I can help you build a product, feature or website. Look through some of my works. If you 
+          like what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
+      </div>
+
+      <hr class="white">
+
+      <div class="skills-div">
+        <div class="language">
+          <p>Language</p>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+
+        <div class="frameworks">
+          <p>Frameworks</p>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+
+        <div class="skills">
+          <p>Skills</p>
+          <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+
+      <hr class="white">
+
+      <div class="media">
+        <a href="https://www.github.com/MindPrepDev">
+          <img src="./images/icons/github.png" alt="connect via github" class="git-img">
+        </a>
+        <a href="https://www.linkedin.com/in/oluwatoyin-olaoye-82880363/">
+          <img src="./images/icons/linked_in.png" alt="connect on linkedin" class="linkedin-img">
+        </a>
+        <a href="https://www.twitter.com/olaoyeelijah">
+          <img src="./images/icons/twit.png" alt="connect via twitter" class="twitter-img">
+        </a>
+        <a href="https://medium.com/@mindprepopps">
+          <img src="./images/icons/meduim.png" alt="meetup on medium" class="medium-img">
+        </a>
+      </div>
```

Comment:
This content should be wrapped in a `<ul>` tag with each icon being wrapped in a `<li>` tag.

<img width="610" alt="Screen Shot 2022-08-18 at 8 23 34" src="https://user-images.githubusercontent.com/82426682/185405722-1c81611e-e338-47ae-848a-a5588a5be7a8.png">

Additionally, you're missing the AngelList icon. While I do understand you may not have a profile on that platform, you still need to include the icon as the primary goal of this project is not to build a ready-to-go portfolio, but rather to teach you how to make clones of static websites.

Additionally, you're missing the button at the bottom altogether, please add it.

<img width="113" alt="Screen Shot 2022-08-18 at 8 25 42" src="https://user-images.githubusercontent.com/82426682/185406220-118c32dc-063c-4b02-a90b-7ac232435a84.png">

<img width="867" alt="Screen Shot 2022-08-18 at 8 26 23" src="https://user-images.githubusercontent.com/82426682/185406386-6316fa18-57f6-46e4-a072-e7fa19a85092.png">

Reply:
Thanks for your kind recommendations, however, I think from all indications, this is not a button. If you check the background of all statuses of a button, none matches.  In fact, I think it more of a sectional heading of "works section". Also, if it were a button, the arrow down should change to arrow up since other pages have been loaded.
@vechicin What do you see to this, can you please checke the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)?